File: src/main/kotlin/net/namibsun/nuztrack/routes/games/GamesTransfer.kt
package net.namibsun.nuztrack.routes.games

typealias GamesListTO = Map<String, String>

data class GameLocationTO(val name: String, val encounters: List<Int>, val milestones: List<MilestoneTO>)
data class MilestoneTO(val name: String, val image: String, val level_cap: Int)

File: frontend/src/data/games/games.transfer.ts
export interface GameTO {
    title: string,
    key: string
    generation: number
}

export interface GameLocationTO {
    name: string,
    game: GameTO,
    encounters: number[],
    milestones: MilestoneTO[]
}

export interface MilestoneTO {
    name: string,
    image: string,
    level_cap: number,
    location: string
}
